### ISSUE TITLE
Mulitple 'search' fields increase violations of DAP

### ISSUE THREAD

**Issue by ianrosadas** [carbon-design-system/carbon#5575]
## Environment

> Operating system: Mac Catalina 10.15.3

> Browser: Chrome  80.0.3987.122

> Automated testing tool and ruleset: - 

> Assistive technology used to verify: -

## Detailed description

> What version of the Carbon Design System are you using? 
    "@carbon/icons": "^10.8.2",
    "@carbon/icons-vue": "^10.8.2",
    "@carbon/vue": "^2.23.0",

> What did you expect to happen?
Multiple search with carbon desing system using vue. 

> What happened instead?
         <span>Opp Number</span>
                <cv-search class=“size_search” label=“Opp Number” placeholder=“Enter opp  number here” v-model=“oppNumber”>
                </cv-search>
                <cv-search class=“size_search” label=“Opp Name” placeholder=“Enter opp  name here” v-model=“oppname”>
                </cv-search>
                <cv-search class=“size_search” label=“Country” placeholder=“Enter country here” v-model=“country”>
                </cv-search>
                <cv-search class=“size_search” label=“Client” placeholder=“Enter client here” v-model=“client”>
                </cv-search>

> What WCAG 2.1 checkpoint does the issue violate?
Multiple WAI-ARIA 'search' landmask must have unique labels specified with 'aria-label' or 'aria-labelledy'.

## Steps to reproduce the issue

1. Add four fields 'search'
2. Run DAP
3. Found the problem DAP that it's not resolve because four fields 'search'

> Please create a reduced test
>
> - Carbon Desing System with VUE.JS:
>   https://carbondesignsystem.com/tutorial/vue/overview
> - Vue:
>   http://vue.carbondesignsystem.com/?path=/story/components-cvsearch--default

## Additional information

- Screenshots or code 
![dap](https://user-images.githubusercontent.com/12039644/76217680-a024cb80-61f1-11ea-96d6-67a96296cb27.png)

![search](https://user-images.githubusercontent.com/12039644/76217683-a155f880-61f1-11ea-8d3b-cb60a07642a5.png)

- Notes

**Comment by aledavila** [comment]
Hi Ian,
Yes that would give you an error if you don't set unique aria-label names. You must pass in unique labels. Thanks
